### PR TITLE
added debian support

### PR DESCRIPTION
**Pull request by tomzo**
When on debian install packages from official glusterfs repository.
Tested debian as a client

### CODE REVIEW

File: recipes/repository.rb [hunk 1]
```diff
@@ -19,6 +19,19 @@
 #
 
 case node['platform']
+when 'debian'
+  include_recipe 'apt::default'
+
+  apt_repository "glusterfs-#{node['gluster']['version']}" do
+    uri "http://download.gluster.org/pub/gluster/glusterfs/#{node['gluster']['version']}/LATEST/Debian/wheezy/apt"
```
Comment (shortdudey123) [inline recipes/repository.rb]: You can't assume `wheezy`

File: recipes/repository.rb [hunk 2]
```diff
@@ -19,6 +19,19 @@
...
+    distribution node['lsb']['codename']
+    components ['main']
+    key "http://download.gluster.org/pub/gluster/glusterfs/#{node['gluster']['version']}/LATEST/Debian/wheezy/pubkey.gpg"
```
Comment (shortdudey123) [inline recipes/repository.rb]: `wheezy` here too

**Comment by shortdudey123** [comment]
Can you add the versions of debian that you have tested this on to the [readme](https://github.com/shortdudey123/chef-gluster#platforms)? will merged after that

**Comment by Seth-Karlo** [comment]
Please also add Debian to the kitchen.yml file

**Comment by shortdudey123** [comment]
That too
@Seth-Karlo i think that will cause merge conflicts for your PR so you probably will need to work w/ that assuming this gets merged first

**Comment by tomzo** [comment]
I can add it to readme, but I do not use vagrant+virtualbox for testing. So I would not be able to verify if tests in kitchen are actually worth anything.

**Comment by Seth-Karlo** [comment]
@shortdudey123 I don't think this will cause any merge conflicts, it's all new code. But if it causes an issue I will fix it.

**Comment by shortdudey123** [comment]
@Seth-Karlo cool

**Pull request merged**